clients/src/test/java/org/apache/kafka/common/protocol/types/ProtocolSerializationTest.java

```diff
@@ -231,6 +231,17 @@ public void testReadNegativeBytesSize() {
         }
     }
 
+    @Test
+    public void testToString() {
+        try {
+            String stuctStr = this.struct.toString();
+            assertFalse("Stuct string should not be null.", stuctStr == null);
+            assertFalse("Stuct string should not be empty.", stuctStr.isEmpty());
+        } catch (Throwable e) {
+            fail("Struct.toString throws an exception");
+        }
```

Comment:
Is it really worth having this catch here? I think it's best to just let the exception propagate. Any method under test can throw an unknown exception after all.